makefiles/common.mk

```diff
@@ -123,6 +123,9 @@ _THEOS_HAS_STAGING_LAYOUT := $(call __exists,$(THEOS_LAYOUT_DIR))
 endif
 
 _THEOS_LOAD_MODULES := $(sort $(call __schema_var_all,,MODULES) $(THEOS_AUTOLOAD_MODULES))
+ifneq ($(THEOS_PACKAGE_SCHEME),)
+_THEOS_LOAD_MODULES += schemes/$(THEOS_PACKAGE_SCHEME)
+endif
```

Comment:
I wonder whether we should deprecate explicitly setting `THEOS_PACKAGE_SCHEME` in favour of `MODULES = <scheme>`

Reply:
I debated that as well. My current thought process is that schemes are modules, but modules aren't necessarily schemes, so keeping the associated variables separate continues that trend. Also helps make managing the necessary changes to internal Theos processes (e.g., stage) easier as we know the state THEOS_PACKAGE_SCHEME is either a valid scheme or blank.

Reply:
Actually, now that I think about it, the stage changes I propose rely on `THEOS_PACKAGE_INSTALL_PREFIX`, not `THEOS_PACKAGE_SCHEME`. The only instance where we seem to rely on `THEOS_PACKAGE_SCHEME` internally is when we modify the linker's library search paths.